homework/index.js

```diff
@@ -28,14 +28,53 @@
     });
     return elem;
   }
-
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
-  }
+    const li = createAndAppend('li', ul, { class: 'item' });
+    const table = createAndAppend('table', li);
+
+    // add Repository name
+    const row1 = createAndAppend('tr', table);
+    const repoName = createAndAppend('th', row1);
+    repoName.innerText = 'Repository:';
```

Comment:
You can set the text content directly with the `createAndAppend()` function like this (it also allows you to get rid of the variable):

```js
createAndAppend('th', row, { text: 'Repository:' });
```

Reply:
done :+1: